src/build-time-render/BuildTimeRender.ts

```diff
@@ -69,6 +69,14 @@ function genHash(content: string): string {
 		.substr(0, 20);
 }
 
+function normalizePath(path: string) {
+	return path
+		.replace(/#.*/, '')
+		.replace(/\/$/, '')
+		.replace(/^\//, '')
+		.toLowerCase();
+}
+
 const trailingSlash = new RegExp(/\/$/);
 const leadingSlash = new RegExp(/^\//);
```

Comment:
These were the regexs I thought could be re-used - or maybe replaced by the normalizePath else where

Reply:
Ah! Good catch. Let me see if I can use normalizePath in other places and clean this up some 👍